lib/dotcom_web/controllers/trip_plan_controller.ex

```diff
@@ -573,6 +573,18 @@ defmodule DotcomWeb.TripPlanController do
     end)
   end
 
+  defp chunk_subway_legs({leg, _idx}) do
+    highest_fare =
+      leg
+      |> Fares.get_fare_by_type(:highest_one_way_fare)
```

Comment:
Not sure why it didn't, but Credo should be flagging the use of pipe here. We can just call the function since there is only one pipe.